### ISSUE TITLE
Where is Server Control Panel ?

### ISSUE THREAD

**Issue by kilohaty** [CompuIves/codesandbox-client#1336]
I can't find Server Control Panel, where is it?

**Comment by CompuIves** [comment]
Hey!

You should own the sandbox and the sandbox should be a container sandbox (it's in the list of server sandboxes) to see the Server Control Panel on the left. Which sandbox are you looking at?

**Comment by kilohaty** [comment]
Thanks!
Oh, I'm import from GitHub, 
is it impossible to use the Server Control Panel?

**Comment by kilohaty** [comment]
Thank you for the help,
I added this file to the project, but there's still no Server Control Panel.
Must the sandbox to be a container sandbox? and how to be a container sandbox?

**Comment by chidimo** [comment]
This issue has been closed but I still can't manage to find the Server Control Panel. The documentation doesn't seem to help.

**Comment by lbogdan** [comment]
Hey @chidimo ,

You can only access the Server Control Panel in server sandboxes that you own, from the side-bar on the left. You can easily test this by forking the [`node template`](https://codesandbox.io/s/node).

![image](https://user-images.githubusercontent.com/2083930/94998162-6ade3180-05b8-11eb-92e9-b7524bb98625.png)

**Comment by chidimo** [comment]
Thanks for replying. I found this issue <https://github.com/codesandbox/codesandbox-client/issues/2111> and I followed the suggestion to create a Github repo and add the `sandbox.config.json`. I used the below config

```json
{
    "template": "node",
    "infiniteLoopProtection": true,
    "hardReloadOnChange": true,
    "view": "browser"
  }
```

Now the control panel is enabled.

I did try to add the config to an existing sandbox but after a few minutes the sandbox would hand and I had to close the tab.

Thanks for this.
